scripts/undefined_blacklist.txt

```diff
@@ -0,0 +1,3 @@
+# This two problems seem related
+src:/scratch/source/trilinos/release/packages/stk/stk_mesh/*
+src:/usr/bin/../lib/gcc/x86_64-linux-gnu/5.4.0/../../../../include/c\+\+/5.4.0/bits/*
```

Comment:
If we have instructions for building docker container (do we?), we should probably add a note to update this when the compiler version changes.